packages/components/src/styles/global.js

```diff
@@ -1,22 +1,26 @@
 import { GlobalStyles } from "twin.macro";
 import React from "react";
+import arimoBoldWoff from "./fonts/arimo-bold.woff";
+import arimoBoldWoff2 from "./fonts/arimo-bold.woff2";
+import arimoRegularWoff from "./fonts/arimo-regular.woff";
+import arimoRegularWoff2 from "./fonts/arimo-regular.woff2";
 import { createGlobalStyle } from "styled-components";
 
 const GlobalFonts = createGlobalStyle`
   @font-face {
     font-family: 'Arimo';
     src:
-      url(fonts/arimo-regular.woff2) format('woff2'),
-      url(fonts/arimo-regular.woff) format('woff');
+      url(${arimoRegularWoff2}) format('woff2'),
+      url(${arimoRegularWoff}) format('woff');
```

Comment:
Follows https://stackoverflow.com/a/55242731

Reply:
Interesting 🤔 . What's the imported value of the woff files? Is it a URL, or just the name of the font?

Reply:
great question, this is what I see when logging:
```
static/media/arimo-regular.f279a33d.woff
static/media/arimo-regular.286d6967.woff2
static/media/arimo-bold.9fd2270a.woff
static/media/arimo-bold.3d3529a4.woff2
```
so looks like a derived URL?

Reply:
Cool, okay, that makes sense. Thanks for looking into it!

Reply:
Is that true for just the compiled version as well? `dist/styles/global.js`